src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -36,7 +36,7 @@ public class DbOnTheRocks : IDbWithSpan
 
     private static readonly ConcurrentDictionary<string, RocksDb> _dbsByPath = new();
 
-    private bool _isDisposed;
+    private int _isDisposed;
```

Comment:
Could we please have `_isDisposed` renamed to `_disposalFlag` or similar? Otherwise, that's a bit misleading as it's no longer a boolean.